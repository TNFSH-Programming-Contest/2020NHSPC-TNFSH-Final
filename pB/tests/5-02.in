4
26.398 990.001
-427.943 990.001
-427.943 -483.961
26.398 -483.961
-337.428 707.521
