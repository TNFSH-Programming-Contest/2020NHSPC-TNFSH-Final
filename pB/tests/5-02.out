4875747.760